pkg/storer/debug.go

```diff
@@ -30,10 +31,11 @@ type CacheStat struct {
 }
 
 type ReserveStat struct {
```

Comment:
It would be good if this is reflected in the OpenAPI spec.

Reply:
we dont have an entry in openAPI for /debugstore

Reply:
If it is not a temporary API, it should be documented in the OpenAPI specification.